client/app/pages/dashboards/hooks/useDashboard.js

```diff
@@ -154,6 +154,26 @@ function useDashboard(dashboardData) {
     [refreshing, loadDashboard]
   );
 
+  const saveDashboardParameters = useCallback(() => {
+    const parameterValues = {};
+    globalParameters.forEach(param => {
+      parameterValues[param.name] = param.value;
+    });
+
+    const currentDashboard = dashboardRef.current;
+
+    return updateDashboard({
+      options: {
+        ...currentDashboard.options,
+        parameterValues: parameterValues,
```

Comment:
I think it is nice to interface compatible with `queries`.
So, how about using `parameters` array as `queries.options.parameters` like:
```
options: {
    ...currentDashboard.options,
    parameters: map(data.options.parameters, p => globalParameters.toSaveableObject()),
}
```